pymel/core/__init__.py

```diff
@@ -6,7 +6,6 @@
 import sys
 import pymel as _pymel
 _pymel.core = sys.modules[__name__]
-import pymel.versions as _versions
```

Comment:
this line wasn't used.  since it's private I don't see why we should keep it, unless we're being super conservative